data-raw/data-checks/stream_team_review/data_checks_template.Rmd

```diff
@@ -0,0 +1,549 @@
+  filter(site == "okie dam", subsite != "okie dam fyke trap") |> 
+  pull(date)
+
+weekly_edi_catch2 <- weekly_edi_catch2_prep |> 
+  mutate(butte_fyke_filter = case_when(site == "okie dam" & as_date(date) %in% okie_rst ~ "rst & fyke",
+                                       site == "okie dam" & !as_date(date) %in% okie_rst ~ "fyke only",
+                                       T ~ "not butte")) |> 
+  filter((species %in% c("chinook", "chinook salmon") | is.na(species)),
+         life_stage != "adult",
+         remove == "keep",
+         butte_fyke_filter != "fyke only") |> 
+  group_by(stream, site, week, year) |> 
+  summarize(count = sum(count, na.rm = T))
+
+weekly_rst_catch <- SRJPEdata::rst_catch |> 
+  mutate(week = week(date),
+         year = year(date)) |> 
+  group_by(stream, site, week, year) |> 
+  summarize(count = sum(count, na.rm = T))
+
+catch_compare_1 <- full_join(weekly_rst_catch |> 
+                               rename(srjpedata_count = count),
+                             weekly_edi_catch1) |> 
+   mutate(p_diff = ((abs(count - srjpedata_count))/((count + srjpedata_count)/2)) * 100)
+
+catch_compare_2 <- full_join(SRJPEdata::weekly_juvenile_abundance_catch_data |> 
+                               select(year, week, stream, site, count) |> 
+                               rename(srjpedata_count = count),
+                             weekly_edi_catch2) |> 
+   mutate(p_diff = ((abs(count - srjpedata_count))/((count + srjpedata_count)/2)) * 100)
+```
+
+
+## Weekly counts {.tabset}
+
+```{r, include = F}
+# 1. rst_catch v EDI data
+# 2. weekly_juvenile_abundance_catch_data v EDI data
+
+p_diff_summary_plot <- function(data_select, stream_select) {
+  data_select |> 
+  filter(stream == stream_select) |> 
+  ggplot(aes(x = week, y = p_diff, color = site)) +
+  geom_point() +
+  scale_color_manual(values = colors_full) +
+  facet_wrap(~year) +
+  theme_bw() +
+  labs(y = "percent difference")
+}
+
+p_diff_greater_than_2 <- function(data_select, stream_select) {
+  data_select |> 
+  filter(stream == stream_select, p_diff > 2) |> 
+  ggplot(aes(x = week, y = p_diff, color = site)) +
+  geom_point() +
+  scale_color_manual(values = colors_full) +
+  facet_wrap(~year) +
+  theme_bw() +
+  labs(y = "percent difference")
+}
+
+na_plot <- function(data_select, site_select) {
+  data_select |> 
+    filter(site== site_select, (is.na(srjpedata_count) & !is.na(count)) | (!is.na(srjpedata_count) & is.na(count))) |> 
+    pivot_longer(cols = c(srjpedata_count, count), names_to = "count_type", values_to = "value") |> 
+  ggplot(aes(x = week, y = value, color = count_type)) +
+  geom_point() +
+  scale_color_manual(values = colors_full) +
+  facet_wrap(~year) +
+  theme_bw() +
+  labs(y = "count")
+}
+```
+
+### Battle/Clear
+
+#### Summary of the percent difference between data on EDI and data used for juvenile abundance model
+
+```{r, echo = F, warning = F, message = F}
+p_diff_summary_plot(catch_compare_2, "battle creek")
```

Comment:
plots could also show monitoring weeks instead of calander weeks but also not nessicary

Reply:
i am going to leave as is.